src/NuGet.Core/NuGet.Commands/SourcesCommands/SourcesRunner.cs

```diff
@@ -0,0 +1,317 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Linq;
+using NuGet.Common;
+
+namespace NuGet.Commands
+{
+    public enum SourcesListFormat
```

Comment:
It this a duplicate of `SourcesListFormat.cs`?

Reply:
Yes. Unreferenced. Will delete. Thx.